run/column_pp.py

```diff
@@ -0,0 +1,51 @@
+import glob
+
+import matplotlib.pyplot as plt
+import netCDF4
+import numpy as np
+
+first = True
+for file in sorted(glob.glob('diagnostic.*.nc')):
+
+    root = netCDF4.Dataset(file, "r", format="NETCDF4")
+
+    time_var = root.groups['metadata'].groups["time"].variables["time"]
+    np_time = np.datetime64(netCDF4.num2date(time_var[:], time_var.units).isoformat())
+
+    if first:
+        time = np.array(np_time)
+        hice = np.array(root.groups["data"].variables["hice"][0].data)
+        hsnow = np.array(root.groups["data"].variables["hsnow"][0].data)
+        tice = np.array(root.groups["data"].variables["tice"][0].data)
+        first = False
+    else:
+        time = np.append(time, np_time)
+        hice = np.append(hice, root.groups["data"].variables["hice"][0].data)
+        hsnow = np.append(hsnow, root.groups["data"].variables["hsnow"][0].data)
+        tice = np.vstack((tice, root.groups["data"].variables["tice"][0].data))
```

Comment:
This could potentially be tidied up by using the `xarray` module. `xarray` works well in python and with netcdf files. Could make reading and plotting of datasets slightly easier (see [here](https://docs.xarray.dev/en/stable/user-guide/io.html#reading-and-writing-files))

Reply:
I've rewritten this post-processing script for Tim's new outputs. It's much simpler now, and I think it works ok without ``xarray``. But I'll keep ``xarray`` in mind for the future.

I also realised that my test description is incomplete, so I've added a bit about the post-processing script.

Reply:
I've opened issue #431 to describe the improvements still needed for a trivial comparison of the test results with figures from the literature.